sim/mcuboot-sys/build.rs

```diff
@@ -81,6 +82,64 @@ fn main() {
         panic!("mcuboot does not support more than one sig type at the same time");
     }
 
+    if psa_crypto_api {
+        if sig_ecdsa || enc_ec256 || enc_x25519 || enc_aes256_ec256 || sig_ecdsa_mbedtls || enc_aes256_x25519 || enc_kw  || enc_aes256_kw {
```

Comment:
I believe this condition shall be updated in the future with enabling new algorithms with the PSA API in the simulator environment.

Reply:
Is this anything which would require changing in this PR? Should I put a comment on it to signpost this, or will it just be as simple as changes being done in the future?

Reply:
No, I nothing needs to be done here, sorry.